cmd/opentelemetry/app/defaultconfig/default_config_test.go

```diff
@@ -59,6 +63,41 @@ func TestService(t *testing.T) {
 				},
 			},
 		},
+		{
+			cfg: ComponentSettings{
+				ComponentType: Collector,
+				StorageType:   "badger",
+			},
+			service: configmodels.Service{
+				Extensions: []string{"health_check"},
+				Pipelines: configmodels.Pipelines{
+					"traces": &configmodels.Pipeline{
+						InputType:  configmodels.TracesDataType,
+						Receivers:  []string{"otlp", "jaeger"},
+						Processors: []string{"batch"},
+						Exporters:  []string{"jaeger_badger"},
+					},
+				},
+			},
+		},
+		{
+			cfg: ComponentSettings{
+				ComponentType: Agent,
+			},
+			service: configmodels.Service{
+				Extensions: []string{"health_check"},
+				Pipelines: configmodels.Pipelines{
+					"traces": &configmodels.Pipeline{
+						Name:       "traces",
+						InputType:  configmodels.TracesDataType,
+						Receivers:  []string{"otlp", "jaeger"},
+						Processors: []string{"batch", "queued_retry"},
```

Comment:
hmm qretry should not be there.

Reply:
`queued_retry` is in here b/c of the missing `testdata/addqueuedprocessor.yaml`.  See my recent push.